### DISCUSSION TITLE
How many decimal places for support in the result set of fpgrowth?

### DISCUSSION THREAD

**Discussion by hpconnox** [rasbt/mlxtend#1097] [Q&A] [answered]
As far as I can say, the support value in the result set (fpgrowth) is returning a maximum of 6 decimal places. Is there any posibility to increase the number of decimal places (to 8 or 10).

**Comment by shenxiangzhuang** [comment, marked as answer]
## TLDR
You can set the `display.precision` by  `pd.set_option("display.precision", 8)` before you print the DataFrame and you will get what you need.


## The details
There is no round operation in source code:

https://github.com/rasbt/mlxtend/blob/63a2655d864c208e1239e04016fc3390bc27ba75/mlxtend/frequent_patterns/fpcommon.py#L64-L78

So it's just the display setting of pandas, you check [pandas doc](https://pandas.pydata.org/docs/reference/api/pandas.set_option.html#pandas.set_option) here.
